variant_calling_benchmarks/common.py

```diff
@@ -28,7 +28,10 @@ def extract_loci_string(patient, variant_filenames):
 
 def add_common_run_args(parser):
     parser.add_argument("configs", nargs="+", help="JSON config files")
-    parser.add_argument("--guacamole-jar", required=True)
+    parser.add_argument("--guacamole-jar", required=True,
```

Comment:
similar to the above, should just make this take a path to a guac directory, and have the script figure it out, no need to ship arcana about our jar-packaging through the cmdline APIs of downstream tools